tasks/release/GitHubController.php

```diff
@@ -32,6 +35,12 @@ public function __construct($username, $token, $project) {
       ],
       'base_uri' => "https://api.github.com/repos/mailpoet/$github_path/",
     ]);
+    $this->http_client_no_base = new Client([
```

Comment:
Actually not needed since Guzzle will ignore base in a request where you pass full absolute URL.